src/main/java/pro/sky/employees/DepartmentController.java

```diff
@@ -0,0 +1,38 @@
+package pro.sky.employees;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/departments")
+public class DepartmentController {
+    private final DepartmentServiceImpl departmentService;
```

Comment:
```suggestion
    private final DepartmentService departmentService;
```
инъекцию надо делать по интерфейсу, а не реализации